.circleci/config.yml

```diff
@@ -345,15 +388,6 @@ jobs:
       - go_build:
```

Comment:
Someone less experienced with CI (me :smile: :disappointed:) added this to make tests work that required the go-filecoin binary to be in the repo, and [added this](https://github.com/filecoin-project/go-filecoin/pull/2609/files#diff-1d37e48f9ceff6d8030570cd36286a61R384) to satisfy tests that require the go-filecoin binary to be in `$GOPATH/bin`. I think we could shave 20-40sec off our build time by replacing this extra build step with a copy command. I believe this pattern is also [used here](https://github.com/filecoin-project/go-filecoin/pull/2609/files#diff-1d37e48f9ceff6d8030570cd36286a61R231).